docs/README.md

```diff
@@ -143,7 +161,30 @@ bazel run //tools:setup_presubmit_repos -- --module <module_name>@<version>
 
 Then follow the instructions to run the build locally.
 
-## Yank a module version
+## Approval and submission
+
+To be submitted, a PR needs to:
+
+- Be approved by a module maintainer and/or a BCR maintainer
+  - When a PR is opened, the `bazel-io` bot will `@`-ping all module maintainers (who do not have `do_not_notify` set) and ask for a review.
+  - Note that a module maintainer can approve a PR by using the normal GitHub PR review flow, despite not having write access to the git repository; the `bazel-io` bot account will approve the PR for merge later.
+- Pass [presubmit](#presubmit) checks
+  - If you see your presubmit check stuck on "blocked", a BCR maintainer needs to explicitly unblock the presubmit run or apply the `presubmit-auto-run` label to your PR. This is to avoid abuse of our CI system. Feel free to ping `@bazelbuild/bcr-maintainers` if you're blocked on this.
+- Pass certain other checks, especially for first-time contributors, such as CLA signing or GitHub workflows that require approval from BCR maintainers.
+
+## Module versions
+
+Bazel has a diverse ecosystem, and projects use a variety of versioning schemes. Bazel modules have a fairly relaxed [version format](https://bazel.build/external/module#version_format), which covers most version strings used by open-source projects. Thus, modules submitted to the BCR are generally versioned according to their upstream project's versions.
+
+### Add-only
+
+To ensure reproducibility, the BCR is add-only; that is, existing versions of a module cannot be modified. If an existing module version needs a fix, it should be fixed upstream and a new version can be submitted. If, however, the fix is only in patch files present in BCR (as in, there is nothing to fix upstream), the convention is to append a `.bcr.<N>` suffix for the new version. For example, if `foo` version `1.2.3` needs patches to fix a problem, you can submit a new version `1.2.3.bcr.1`.
+
+### Pre-releases
+
+If upstream hasn't released a new version in a long time (for example, due to project owner inactivity), but you'd still like to submit a version based on a main branch commit, you can use the `-bcr.<N>` suffix (note the `-` to denote a pre-release). For example, if `foo`'s current version is `1.19.0`, you can submit a new version `1.20.0-bcr.1`.
```

Comment:
I think that [pseudo-versions](https://go.dev/ref/mod#pseudo-versions) work better for this as they make it more obvious that the version comes from a commit rather than a tag. It also combines well with the `.bcr.<N>` convention (rather than reusing it for a different purpose).

Reply:
sounds good! done.